### ISSUE TITLE
No Add button on Deployment -> Deployment tasks with Firefox 10ESR

### ISSUE THREAD

**Issue by ddurieux** [fusioninventory/fusioninventory-for-glpi#772]
---

Author Name: **James Pulver** (James Pulver)
Original Redmine Issue: 1463, http://forge.fusioninventory.org/issues/1463
Original Date: 2012-02-13
Original Assignee: Gonéri Le Bouder

---

When you go to Deploy -> Deployment Tasks, add tasks - there's no add button to add the task after you fill in details. This works from the FusionInventory -> Task Management tab though.

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **Gonéri Le Bouder** (Gonéri Le Bouder)
Original Date: 2012-02-13T15:08:23Z

---

What browser do you use?

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **Gonéri Le Bouder** (Gonéri Le Bouder)
Original Date: 2012-02-13T15:09:42Z

---

I've just tested with the Firefox 10 and MSIE9 and it fine.

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **Gonéri Le Bouder** (Gonéri Le Bouder)
Original Date: 2012-02-14T21:33:26Z

---

```
CREATE VIEW `glpi_plugin_fusinvdeploy_tasks`
AS SELECT * FROM glpi_plugin_fusioninventory_tasks;


CREATE VIEW `glpi_plugin_fusinvdeploy_taskjobs`
AS SELECT `id`,
`plugin_fusioninventory_tasks_id` AS `plugin_fusinvdeploy_tasks_id`,
`entities_id`, `name`, `date_creation`, `retry_nb`,
`retry_time`, `plugins_id`, `method`, `definition`,
`action`, `comment`, `users_id`, `status`,
`rescheduled_taskjob_id`, `statuscomments`,
`periodicity_count`, `periodicity_type`, `execution_id`
FROM glpi_plugin_fusioninventory_taskjobs;

```

These two SQL statements where not processed during the installation.

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **Gonéri Le Bouder** (Gonéri Le Bouder)
Original Date: 2012-04-27T19:53:24Z

---

fixed during the 0.80+1.2 release cycle.
